bids/grabbids/bids_layout.py

```diff
@@ -9,7 +9,9 @@
 from os.path import join as pathjoin
 
 from .bids_validator import BIDSValidator
+from .utils import merge_rowise
 from grabbit import Layout
+import pandas as pd
```

Comment:
We don't want to make pandas a dependency of grabbids just for event merging. I would suggest either moving the merging functionality into a separate utility (e.g., in `bids.utils`), or moving the import inside `get_events` (and conditioning on the user selecting `return_type='df'`).